src/qcodes/instrument_drivers/AimTTi/_AimTTi_PL_P.py

```diff
@@ -85,23 +93,26 @@ def __init__(
             "Here, the integer 1 is for the Low range, "
             "and integer 2 is for the High range.",
         )
+        """Set the current range of the output.Here, the integer 1 is for the Low range, and integer 2 is for the High range."""
```

Comment:
Missing space
```suggestion
        """Set the current range of the output. Here, the integer 1 is for the Low range, and integer 2 is for the High range."""
```

Reply:
Yes. this is a typo in the original docstring too so should also be fixed there